plasmapy/formulary/parameters.py

```diff
@@ -1349,7 +1374,7 @@ def plasma_frequency(n: u.m ** -3, particle: Particle, z_mean=None) -> u.rad / u
 
     omega_p = u.rad * Z * e * np.sqrt(n / (eps0 * m))
 
-    return omega_p.si
+    return omega_p.to(u.rad / u.s)
```

Comment:
Note, you don't need to add the `.to(u.rad / u.s)` since `@validate_quantities` will take care of this.  In fact, `@validate_quantities` exists because we don't want to have to repeatedly do this.